### PR TITLE
Make transition property private.

### PR DESCRIPTION
**Pull request by ebidel**
R: @blasten

### CODE REVIEW

File: paper-drawer-panel.html [hunk 1]
```diff
@@ -207,7 +207,7 @@
         left: 0;
         visibility: hidden;
         opacity: 0;
-        transition: opacity ease-in-out 0.38s, visibility ease-in-out 0.38s;
+        app=header: opacity ease-in-out 0.38s, visibility ease-in-out 0.38s;
```
Comment (blasten) [inline paper-drawer-panel.html]: typo?

Comment (ebidel) [inline paper-drawer-panel.html]: Yes :( Fixing

**Comment by ebidel** [comment]
rebased

**Comment by blasten** [comment]
LGTM

**Pull request merged**